### ISSUE TITLE
Request to train models on higher depth resolution

### ISSUE THREAD

**Issue by kaelsonofkrypto** [DepthAnything/Depth-Anything-V2#32]
Hi, is it possible that you all can further train the v2 models on higher depth resolutions than 518. So that v2 can properly render depth maps that use high depth resolutions above 518. I  believe being able to achieve a minimum depth resolution of 1400 but more specifically 2000 would get rid of a lot of the artifact issues with any depth, but u all would need to train on high depth resolution. currently if you use a depth resolution above 518 it instantly degrades the depth the higher u go and i understand this is a limit because thats what v2 was trained on. So the only solution would be to retrain it on higher resolutions. Right now on all of the models i have experienced a wobble jiggle artifact flickering on background. This even occurs on any large v2. This model has more detail v1 but it has less stability compared to v1

**Comment by LiheYoung** [comment]
We do not have enough computational resources for higher-resolution training. Besides, as far as I know, it is impractical (or not trivial) to train the ViT-G model (1.3B parameters) under the 1500+ resolution, with A100 GPUs.

**Comment by kaelsonofkrypto** [comment]
th

> We do not have enough computational resources for higher-resolution training. Besides, as far as I know, it is impractical (or not trivial) to train the ViT-G model (1.3B parameters) under the 1500+ resolution, with A100 GPUs.

the reason i asked is because i was able to achieve very high quality 3d near artifact free at 1400 on any s, however the depth map was inaccurate because the model werent trained at that level, some areas of depth when accurate, the problem at 518 there is a flicker artifact that occurs that doesnt occur at high depth resolutions like 1400

**Comment by hit2sjtu** [comment]
> We do not have enough computational resources for higher-resolution training. Besides, as far as I know, it is impractical (or not trivial) to train the ViT-G model (1.3B parameters) under the 1500+ resolution, with A100 GPUs.

I am wondering how many gpu (and days) are used to train depth anything v2?

**Comment by Byeongjun1022** [comment]
> Hi, thank you for pushing progress with any depth. Is it possible for you all to further train the v2 models to handle higher depth resolutions beyond 518? Achieving a minimum depth resolution of 1400, ideally 2000, would significantly reduce artifact issues. Currently, using resolutions above 518 degrades depth quality due to v2's training limitations. Retraining on higher resolutions seems necessary to address this. I've noticed a wobble/jiggle artifact flickering in the background on all models, including any large v2. Despite the v2 models increased detail compared to v1. V2 has less stability than the v1 models and a lot of flickering issues. It also does NOT handle mirror and reflections well.

Can I know the meaning of "wobble/jiggle artifact flickering in the background on all models"?
Do you mean that the flickering is induced when applied to high-resolution video? or jiggle artifact occurs when applied to high-resolution image?

**Comment by kaelsonofkrypto** [comment]
actu

> > Hi, thank you for pushing progress with any depth. Is it possible for you all to further train the v2 models to handle higher depth resolutions beyond 518? Achieving a minimum depth resolution of 1400, ideally 2000, would significantly reduce artifact issues. Currently, using resolutions above 518 degrades depth quality due to v2's training limitations. Retraining on higher resolutions seems necessary to address this. I've noticed a wobble/jiggle artifact flickering in the background on all models, including any large v2. Despite the v2 models increased detail compared to v1. V2 has less stability than the v1 models and a lot of flickering issues. It also does NOT handle mirror and reflections well.
> 
> Can I know the meaning of "wobble/jiggle artifact flickering in the background on all models"? Do you mean that the flickering is induced when applied to high-resolution video? or jiggle artifact occurs when applied to high-resolution image?

actually its the opposite, the lower the depth resolution the more jiggle wobble flickering happens, the higher the depth resolution the less flickering happens. the problem however is even tho high depth resolution helps the wiggle jiggle wobble flickering, the depth map gets degraded because any depth wasnt trained on anything higher than 518 depth map. This is why i am asking for training on higher resolutions because higher resolutions help reduce the flickering even more. the 518 is too low, too much flickering happens at this level

**Comment by Byeongjun1022** [comment]
> actually its the opposite, the lower the depth resolution the more jiggle wobble flickering happens, the higher the depth resolution the less flickering happens. the problem however is even tho high depth resolution helps the wiggle jiggle wobble flickering, the depth map gets degraded because any depth wasnt trained on anything higher than 518 depth map. This is why i am asking for training on higher resolutions because higher resolutions help reduce the flickering even more. the 518 is too low, too much flickering happens at this level. also edge artifacting is significantly reduced at higher depth resolutions, the only problem is really the depth maps looses its gamma and contrast becoming degraded and thats only because it was never trained higher than 518.

Thanks for reply. So, the flickering is induced by low-resolution video? Do I understand correctly?

**Comment by kaelsonofkrypto** [comment]
> > actually its the opposite, the lower the depth resolution the more jiggle wobble flickering happens, the higher the depth resolution the less flickering happens. the problem however is even tho high depth resolution helps the wiggle jiggle wobble flickering, the depth map gets degraded because any depth wasnt trained on anything higher than 518 depth map. This is why i am asking for training on higher resolutions because higher resolutions help reduce the flickering even more. the 518 is too low, too much flickering happens at this level. also edge artifacting is significantly reduced at higher depth resolutions, the only problem is really the depth maps looses its gamma and contrast becoming degraded and thats only because it was never trained higher than 518.
> 
> Thanks for reply. So, the flickering is induced by low-resolution video? Do I understand correctly?

yes, that is correct. i have done multiple tests confirming this at low and high depth resolutions and consistently each time i did the 518 depth resolution, it had more flickering than at higher depth resolutions
